### PR TITLE
Edit the manifest file

### PR DESCRIPTION
**Pull request by nabobalis**
Also removed the JOSS paper since it's not for this version anymore.

### CODE REVIEW

**Comment by wtbarnes** [comment]
It is still in the package template I believe. If we don't need it, we should delete it there too.

**Comment by nabobalis** [comment]
Something I found out:

My local dist has all the files if I build it in my environment but if I remove the package data lines from setup.cfg, the dist that tox builds lacks them.

File: setup.cfg [hunk 1]
```diff
@@ -40,7 +40,6 @@ install_requires =
   pandas>=0.23.0
   astropy>=3.2
   parfive[ftp]>=1.1.0
-  importlib_resources;python_version<"3.7"
```
Comment (Cadair) [inline setup.cfg:43]: Are we backporting this?

File: tox.ini [hunk 2]
```diff
@@ -92,7 +91,6 @@ description = Run all style and file checks with pre-commit
 deps =
     pre-commit
 commands =
-    pre-commit install-hooks
```
Comment (Cadair) [inline tox.ini:95]: I added this separately to make it more obvious if an issue occurred during install vs run, also just to be explicit about things. Don't mind if you still want to delete it tho.

File: MANIFEST.in [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
-# Include the docs and examples
-recursive-include docs *
-recursive-include examples *
-prune build
-prune docs/_build
-prune docs/api
-
-# Globally exclude compiled files
-global-exclude *.pyc *.o
+# Exclude specific files
```
Comment (Cadair) [inline MANIFEST.in:1]:
```suggestion
# Exclude specific files
# All files which are tracked by git and not explicitly excluded here are included by setuptools_scm
```

File: tox.ini (hunk 2 above)
Comment (nabobalis) [reply tox.ini:95]: Fair enough, I will add it back.

File: setup.cfg (hunk 1 above)
Comment (nabobalis) [reply setup.cfg:43]: To 2.0?

File: tox.ini [hunk 4]
```diff
@@ -92,7 +91,6 @@ description = Run all style and file checks with pre-commit
...
     pre-commit run --verbose --all-files --show-diff-on-failure
```
Comment (nabobalis) [inline tox.ini:94]:
```suggestion
    pre-commit install-hooks
    pre-commit run --verbose --all-files --show-diff-on-failure
```

**Comment by dstansby** [comment]
Is there a good reason to get rid of the JOSS paper? At least if it's being deleted here I think it should be archived somewhere else (that isn't just the git history of the sunpy repo).

**Comment by nabobalis** [comment]
Its stored in both 1.0 and 1.1 branches and releases. Whether that's enough from the sunpy side I am not sure.

File: MANIFEST.in [hunk 5]
```diff
@@ -1,27 +1,22 @@
...
+# All files which are tracked by git and not explicitly excluded here are included by setuptools_scm
+exclude .codecov.yaml
+exclude .mailmap
+exclude .editorconfig
+exclude .gitattributes
+exclude .gitignore
+exclude .pep8speaks.yml
+exclude .pre-commit-config.yaml
+exclude .pre-commit-config.yaml
```
Comment (dstansby) [inline MANIFEST.in:10]:
```suggestion
```
Duplicated

**Comment by dstansby** [comment]
Maybe stick a copy in https://github.com/sunpy/sunpy-1.0-paper under a `joss` folder?

**Comment by nabobalis** [comment]
> Maybe stick a copy in [sunpy/sunpy-1.0-paper](https://github.com/sunpy/sunpy-1.0-paper) under a `joss` folder?

Good idea, I will do that.

File: setup.cfg (hunk 1 above)
Comment (dstansby) [reply setup.cfg:43]: Has this gone because we now depend on >=3.7?

File: tox.ini [hunk 6]
```diff
@@ -28,7 +28,6 @@ changedir = .tmp/{envname}
 #
 description =
     run tests
-    alldeps: with all optional dependencies
```
Comment (dstansby) [inline tox.ini:31]: Why has this gone?

File: setup.cfg [hunk 7]
```diff
@@ -72,13 +71,6 @@ docs =
   astroquery
   jplephem
 
-[options.package_data]
-sunpy = CITATION.rst
-sunpy.data = sunpyrc, test/*, test/*/*, data_manager/tests/db_testdata.csv
-sunpy.database = tests/test_table.txt
-sunpy.io = special/asdf/schemas/sunpy.org/sunpy/*/*.yaml, special/asdf/schemas/sunpy.org/sunpy/*/*/*.yaml
-sunpy.tests = figure_hashes_py36.json
-sunpy.net = vso/data/*, jsoc/data/*
```
Comment (dstansby) [inline setup.cfg:74]: Why have these gone?

**Comment by nabobalis** [comment]
> Maybe stick a copy in [sunpy/sunpy-1.0-paper](https://github.com/sunpy/sunpy-1.0-paper) under a `joss` folder?

The joss paper is now here and added to the readme. https://github.com/sunpy/sunpy-1.0-paper

File: setup.cfg (hunk 7 above)
Comment (nabobalis) [reply setup.cfg:74]: Since we use setuptools_scm, it basically adds all the package data for us that is under the repo.

File: tox.ini (hunk 6 above)
Comment (nabobalis) [reply tox.ini:31]: It was never used.

File: setup.cfg (hunk 1 above)
Comment (nabobalis) [reply setup.cfg:43]: We used to import this in our init file to get the sunpy version, now we don't anymore.

**Comment by nabobalis** [comment]
```

ERROR ../../.tox/py38-online/lib/python3.8/site-packages/sunpy/net/helio/tests/test_helio.py::test_get_table_names - requests.exceptions.ConnectionError: HTTPConnectionPool(host...
ERROR ../../.tox/py38-online/lib/python3.8/site-packages/sunpy/net/helio/tests/test_helio.py::test_select_table - requests.exceptions.ConnectionError: HTTPConnectionPool(host='w...
ERROR ../../.tox/py38-online/lib/python3.8/site-packages/sunpy/net/helio/tests/test_helio.py::test_time_query - requests.exceptions.ConnectionError: HTTPConnectionPool(host='www...
```
Merging.

**Pull request merged**